### PR TITLE
Adding Confidence Score to Server Responses

### PR DESCRIPTION
**Pull request by LouaiZahran**
This is a simple pull request by which the server is now able to respond with the confidence score of the predicted label. This is achieved by using a softmax layer which takes the output of the last layer of the network and normalize it so that the sum of all logits becomes 1.

Example usage using postman:
[
Screenshot 2024-05-07 183252
](url)

### CODE REVIEW

**Review by AbdelrahmanMosly — APPROVED** [review]

**Review by Bazina — APPROVED** [review]
LGTM
